crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -211,19 +212,18 @@ async fn sys_validation_workflow_inner(
     })
 }
 
-// TODO: Some of these params are unnecessary or soon will be:
-// - The Workspace is only needed for some checks which are done in sys validation inappropriately, like fork detection.
-// - The Conductor handle is only needed for another inappropriate check of entry type, which invokes wasm and is not proper sys validation.
-// These two params can go away soon.
-// What's important to note is that the cascade must be passed in explicitly so that it can be mocked.
-async fn validate_op(
+/// TODO: Some of these params are unnecessary or soon will be:
+/// - The Workspace is only needed for some checks which are done in sys validation in appropriately, like fork detection.
```

Comment:
Thanks for pointing this out, I needed to remove these comments because I addressed the issues in the PR. Those checks were inappropriate because they used the DNA wasm, which means they should be considered app validation.

A large part of this PR was refactoring this function to allow sys validation to be tested without an entire real DNA. It had gotten out of hand.